src/java/arjdbc/postgresql/PostgreSQLRubyJdbcConnection.java

```diff
@@ -306,6 +307,24 @@ protected void setArrayParameter(final ThreadContext context,
         statement.setArray(index, connection.createArrayOf(typeName, values));
     }
 
+    protected void setDecimalParameter(final ThreadContext context,
+                                       final Connection connection, final PreparedStatement statement,
+                                       final int index, final IRubyObject value,
+                                       final IRubyObject attribute, final int type) throws SQLException {
+        if (value instanceof RubyBigDecimal) {
+            RubyBigDecimal bigDecimal = (RubyBigDecimal) value;
+
+            // too bad RubyBigDecimal.isNaN() isn't public
+            if (bigDecimal.nan_p(context) == context.runtime.getTrue()) {
```

Comment:
feel free to use the `context.tru`, `fals`, `nil` "shortcuts"

Reply:
ok, will do in the next round